spec/views/post_show_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe 'Post Show', type: :feature do
+  before(:each) do
+    @user1 = User.create(
+      name: 'Doe John',
+      photo: 'http://placeimg.com/640/480/any',
+      bio: 'I am a develpoer',
+      post_counter: 0
+    )
+
+    @user2 = User.create(
+      name: 'Doe makinde',
+      photo: 'http://placeimg.com/640/450/any',
+      bio: 'I am an engineer',
+      post_counter: 0
+    )
+
+    @post = Post.create(
+      author: @user1,
+      title: 'My test post',
+      text: 'This is my test post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+
```

Comment:
- [ ] Please add a test here that asserts you can see the username of each commentor.